iot/manager/manager.go

```diff
@@ -677,7 +753,10 @@ func main() {
 	flag.Parse()
 
 	// Retrieve project ID from console
-	projectID := os.Getenv("GCLOUD_PROJECT")
+	projectID := os.Getenv("GOLANG_SAMPLES_PROJECT_ID")
```

Comment:
I think it's OK to leave this one out and only do the normal ones. The test setup sets the `GOOGLE_CLOUD_PROJECT` variable.